lib/index.js

```diff
@@ -402,20 +402,28 @@ client.on('message', message => {
         // document command in help section
       case 'dotprefix':
         if (isAdmin) {
-          db.updateServerPrefix(
-            serverId, args[0]
-          ).then(data => {
-            serversConfigStore = []
-            if (data) client.guilds.cache.forEach(server => {
-              db.saveServerConfig(
-                +new Date, server.id //rework schema to eliminate the need of pushing date unecessarily here or make a find function
-              ).then(data => {
-                serversConfigStore.push(data)
+          if(args[0] != undefined) {
+            if(args[0].length == 1) {
+             db.updateServerPrefix(
+               serverId, args[0]
+             ).then(data => {
+               serversConfigStore = []
+               if (data) { message.channel.send("Prefix changed."); } if (data) client.guilds.cache.forEach(server => {
```

Comment:
This seems like an unecessary if statement, would be best if message would be shown under 415 line.

Reply:
We should be good to merge this once this part is changed.